app/models/mention_mailer.rb

```diff
@@ -8,8 +9,9 @@ def self.default_url_options
   
   def notify_mentioning(issue, mentioned_by, mentioned_text, mentioned_user)
     @issue = issue
-    @mentioned_by = mentioned_by
+    @mentioned_by = mentioned_user.name
```

Comment:
Yes, they are. This is part of my 3rd patch, to hide redmine usernames, so they would not be a vector for attack.